README.md

```diff
@@ -32,11 +31,11 @@ To get a local copy up and running follow these simple example steps.
 
 ## Visit And Open Files
 
-[Visit Repo](https://github.com/AdornChoga/Javascript_Capstone)
+[Visit Repo](https://github.com/mrparkersson/Calculator)
 
 ## Download Repo
 
-[Download Repo](https://github.com/AdornChoga/Javascript_Capstone)
+[Download Repo](https://github.com/mrparkersson/Calculator)
 
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.